src/pages/new-game/new-game.ts

```diff
@@ -89,6 +150,7 @@ export class NewGamePage {
     if (index > -1) {
       this.players.splice(index, 1);
     }
+    this.updateCardsSelect();
```

Comment:
Why should the be called when the name is changed?

Reply:
It is not, github makes it seem that way in the preview. It is called  in deletePlayer

Reply:
Damn you GitHub, maybe better for me to avoid doing reviews on my phone 😂